app/services/permissions/user.rb

```diff
@@ -0,0 +1,30 @@
+module Permissions
+  module User
+    extend ActiveSupport::Concern
+
+    def can_edit?(org)
+      superadmin? || (!org.nil? && organisation == org)
+    end
+
+    def can_delete?(_org)
+      superadmin?
+    end
+
+    def can_request_org_admin?(org)
+      !superadmin? && organisation != org && pending_organisation != org
```

Comment:
@StlMaris123 to make this more readable, maybe we try this

```suggestion
     !superadmin? && (organisation != org) && (pending_organisation != org)
```